wordpress-mobile/WordPress-iOS
# Blogging Prompts Feature Introduction: update lightbulb icon

Ref: #18176
Depends on: #18304

The lightbulb Material icon is now used in the Blogging Prompts feature introduction header instead of the SF Symbol.

To test:
- Enable `bloggingPrompts` feature flag.
- Run the app.
- When the app launches, the Feature Introduction will appear. 
- Verify the updated lightbulb icon is displayed.

| Before | After |
|--------|-------|
| <img width="443" alt="before" src="https://user-images.githubusercontent.com/1816888/162299644-c51c07ed-5443-48ec-876a-de07ff4d8952.png"> | <img width="438" alt="after" src="https://user-images.githubusercontent.com/1816888/162299629-8014e64e-6a5a-486d-a336-ff7d8710852f.png"> |

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
